packages/projects-docs/pages/learn/plans/_meta.json

```diff
@@ -0,0 +1,8 @@
+{
+  "plan-overview": "Choosing a plan",
+  "permissions": "User Permissions",
+  "preferences": "Preferences",
+  "workspace": "Workspaces",
```

Comment:
shouldn't workspaces and pricing faq be a bit higher in the list? User permissions and Preferences are a bit out of place imo, right after choosing a plan